Packs/NonSupported/.pack-ignore

```diff
@@ -92,4 +92,7 @@ ignore=RP102
 ignore=BA101
 
 [file:reputation-cve_5_4_9.json]
-ignore=RP102
\ No newline at end of file
+ignore=RP102
+
+[file:reputations.json]
+ignore=RP104
```

Comment:
we don't have RP104

Reply:
Its a new error I added in this PR - https://github.com/demisto/demisto-sdk/pull/519/files